### PR TITLE
Support Boba Network contracts

### PR DESCRIPTION
**Pull request by InoMurko**
### What was wrong?

Gnosis is not officially supporting Boba Network, but I'd rather keep on deploying your tags rather than maintain our fork. 
Not sure if this is OK or not?
### How was it fixed?

Add Boba deterministic deployments.

### CODE REVIEW

File: config/settings/local.py [hunk 1]
```diff
@@ -13,11 +13,8 @@
     default="aHdCBMHXuxIxEhfRGFRp7Cp3N9CqEZEEAvwZVlBCazKExkEnzvVs4bYWC8Qqh9lg",
 )
 # https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = [
-    "localhost",
-    "0.0.0.0",
-    "127.0.0.1",
-]
+allowed_hosts = os.getenv("DJANGO_ALLOWED_HOSTS", "localhost,0.0.0.0,127.0.0.1,*")
+ALLOWED_HOSTS = [allowed_host.strip() for allowed_host in allowed_hosts.split(",")]
```
Comment (rmeissner) [inline config/settings/local.py:17]: Could you mention why this change is needed?

Comment (InoMurko) [reply config/settings/local.py:17]: Oh sorry, this slipped.
In a docker compose setup with all needed gnosis infrastructure the transaction service would reject requests coming from other containers.
So I've added the default `*` (allow all) and reading from env var `DJANGO_ALLOWED_HOSTS` in case one would like to do something custom. Similar code exists in  https://github.com/gnosis/safe-transaction-service/blob/master/config/settings/production.py#L9

Comment (Uxio0) [reply config/settings/local.py:17]: Please remove this change from the PR 🙂

**Review by Uxio0 — CHANGES_REQUESTED** [review]
Could you link the PR for BOBA contracts on https://github.com/gnosis/safe-deployments ?

File: config/settings/local.py (hunk 1 above)
Comment (InoMurko) [reply config/settings/local.py:17]: Can I add it in a separate PR?

**Comment by InoMurko** [comment]
> Could you link the PR for BOBA contracts on https://github.com/gnosis/safe-deployments ?

https://github.com/gnosis/safe-deployments/pull/41

**Comment by Uxio0** [comment]
Please rebase master branch 😉

**Review by Uxio0 — APPROVED** [review]
